client/extensions/relatedArticles/src/EventsRelatedArticles/PreviewArticle.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
```

Comment:
We already have an [ArchivePreviewComponent](https://github.com/superdesk/superdesk-planning/blob/develop/client/components/Archive/ArchivePreview/index.tsx), which is used in the Assignment side panel. Could this be used here? They both provide the same functionality, so it's best to stick to DRY principle of re-using an existing component